en/ros2/px4_ros2_navigation_interface.md

```diff
@@ -0,0 +1,303 @@
+
+   ```sh
+   [INFO] [1702030701.836897756] [example_global_navigation_node]: example_global_navigation_node running!
+   [DEBUG] [1702030702.837279784] [example_global_navigation_node]: Successfully sent position update to navigation interface.
+   [DEBUG] [1702030703.837223884] [example_global_navigation_node]: Successfully sent position update to navigation interface.
+   ```
+
+7. In the PX4 shell, you can check that PX4 receives global position updates:
+
+   ```sh
+   listener aux_global_position
+   ```
+
+   The output should look like:
+
+   ```sh
+   TOPIC: aux_global_position
+   aux_global_position
+      timestamp: 46916000 (0.528000 seconds ago)
+      timestamp_sample: 46916000 (0 us before timestamp)
+      lat: 12.343210
+      lon: 23.454320
+      alt: 12.40000
+      alt_ellipsoid: 0.00000
+      delta_alt: 0.00000
+      eph: 0.31623
+      epv: 0.44721
+      terrain_alt: 0.00000
+      lat_lon_reset_counter: 0
+      alt_reset_counter: 0
+      terrain_alt_valid: False
+      dead_reckoning: False
+   ```
+
+8. Now you are ready to use the navigation interface to send your own position updates.
+
+## How to use the Library
+
+To send a position measurement, you populate the position struct with the values you have measured.
+Then call the interface's update function with that struct as the argument.
+
+For a basic example of how to use this interface, check out the [examples](https://github.com/Auterion/px4-ros2-interface-lib/tree/main/examples/cpp/navigation) in the `Auterion/px4-ros2-interface-lib` repository, such as [examples/cpp/navigation/local_navigation](https://github.com/Auterion/px4-ros2-interface-lib/blob/main/examples/cpp/navigation/local_navigation/include/local_navigation.hpp) or [examples/cpp/navigation/global_navigation](https://github.com/Auterion/px4-ros2-interface-lib/blob/main/examples/cpp/navigation/local_navigation/include/global_navigation.hpp).
+
+### Local Position Updates
+
+First ensure that the PX4 parameter [`EKF2_EV_CTRL`](../advanced_config/parameter_reference.md#EKF2_EV_CTRL) is properly configured to fuse external local measurements, by setting the appropriate bits to `true`:
+
+- `0`: Horizontal position data
+- `1`: Vertical position data
+- `2`: Velocity data
+- `3`: Yaw data
+
+To send a local position measurement to PX4:
+
+1. Create a [`LocalPositionMeasurementInterface`](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1LocalPositionMeasurementInterface.html) instance by providing it with: a ROS node, and the pose and velocity reference frames of your measurements.
+2. Populate a [`LocalPositionMeasurement`](https://auterion.github.io/px4-ros2-interface-lib/structpx4__ros2_1_1LocalPositionMeasurement.html) `struct` with your measurements.
+3. Pass the `struct` to the `LocalPositionMeasurementInterface` [`update()`](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1LocalPositionMeasurementInterface.html#a6fd180b944710716d418b2cfe1c0c8e3) method.
```

Comment:
I've inconsistently updated `update()`. When talking about methods I prefer having `()` after the name because then it is obvious you're talking about a method. Some people don't like this because they think the empty bracket indicates "no arguments".